dockerfiles/nexus/Dockerfile

```diff
@@ -4,6 +4,8 @@ MAINTAINER Snyk Ltd
 
 USER root
 
+RUN apt update && apt install -y make python gcc
```

Comment:
where did this come from? did you add this @gitphill ?

Reply:
yep I think we need to re-visit this entire Dockerfile, the base image for broker was changed, we should start again from an existing Dockerfile (artifactory?) to make sure we cover everything that disappears / appeared